core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html

```diff
@@ -21,8 +21,17 @@ <h3>Review translation</h3>
     <div class="oppia-suggestion-review-panel-container float-right">
       <strong> Translation:</strong>
       <div class="oppia-suggestion-review-panel">
-        <angular-html-bind html-data="translationHtml">
-        </angular-html-bind>
+        <div>
+          <angular-html-bind html-data="translationHtml" ng-if="!isEditing">
+          </angular-html-bind>
+          <schema-based-editor schema="HTML_SCHEMA" local-value="editedContent.html" ng-if="isEditing">
+          </schema-based-editor>
+        </div>
+        <div class="oppia-translation-review-edit-buttons">
+          <button class="btn btn-secondary" ng-if="isEditing" ng-click="cancelEdit()">Cancel</button>
+          <button class="btn btn-success" ng-if="isEditing" ng-click="updateTranslate()">Update</button>
+          <button class="btn btn-secondary" ng-if="!isEditing && status === 'review'" ng-click="editTranslate()">Edit</button>
```

Comment:
Can we move the edit button to above the review panel, right-aligned?

Reply:
Done